Sources/NonEmpty/NonEmpty+ExpressibleByArrayLiteral.swift

```diff
@@ -0,0 +1,10 @@
+extension NonEmpty: ExpressibleByArrayLiteral where Collection: ExpressibleByArrayLiteral {
+  public init(arrayLiteral elements: Element...) {
+    assert(!elements.isEmpty, "Cannot initialize a NonEmpty array from an empty literal!")
```

Comment:
Just finessing the wording and switching to be a precondition.

```suggestion
    precondition(!elements.isEmpty, "Can't construct \(Self.self) from empty array literal")
```